monitoring-as-code/src/sli-value-libraries/average-latency-using-seconds-target-metric.libsonnet

```diff
@@ -0,0 +1,115 @@
+// Divides the sum of if target metric samples above latency target by the sum of count over time of
```

Comment:
Slight typo, should this not read
`Divides the count of target metric samples above latency target by the overall count of samples`